executor/runtime/docker/docker_linux.go

```diff
@@ -97,6 +97,14 @@ var systemServices = []serviceOpts{
 		required:     true,
 		enabledCheck: shouldStartServiceMesh,
 	},
+	{
+		humanName: "abmetrix",
+		unitName:  "titus-abmetrix",
+		required:  true,
```

Comment:
Let's make required: false for now? And we can look at logs to see if there are any problems?

Reply:
i set to false.  not entirely sure what that means, but the other logic should turn it off if it doesn't have sufficient config to start up